docs/conf.py

```diff
@@ -98,13 +98,13 @@
 #     'both' - Both the class’ and the __init__ method’s docstring are
 #              concatenated and inserted.
 #     'init' - Only the __init__ method’s docstring is inserted.
-autoclass_content = 'both'
+autoclass_content = 'class'
 
 # This value selects if automatically documented members are sorted
 # alphabetical (value 'alphabetical'), by member type (value
 # 'groupwise') or by source order (value 'bysource'). The default is
 # alphabetical.
-autodoc_member_order = 'bysource'
+autodoc_member_order = 'alphabetical'
```

Comment:
This makes things easier to find. (If you want source order, you can read the source.)